src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -17,7 +36,7 @@
     public Credentials() {
         String homeDir = System.getProperty("user.home");
         String credentialsPath = homeDir + "/.lucidtech/credentials.cfg";
```

Comment:
This might break on Windows. Is there a path separator symbol / library we can use that is built-in?